doc/template.html

```diff
@@ -23,7 +23,20 @@
 
     <div id="column1" data-id="__ID__" class="interior">
       <header>
-        <h1>Node.js __VERSION__ Documentation</h1>
+        <div class="header-container">
+          <h1>Node.js __VERSION__ Documentation</h1>
+          <button class="theme-toggle-btn" id="theme-toggle-btn">
```

Comment:
Adding a `hidden` attribute so folks reading the docs with JS disabled doesn't see an unusable button
```suggestion
          <button class="theme-toggle-btn" id="theme-toggle-btn" hidden>
```